azkaban-db/src/main/sql/create.quartz-tables-all.sql

```diff
@@ -0,0 +1,165 @@
+-- This file collects all quartz table create statement required for quartz 2.2.1
```

Comment:
The current convention is to create one .sql file per table. 
And the database setup code relies on it.

Is there any reason to break this convention?

Reply:
these tables originally reside in one file together. I would believe we should not manage/modify these sql statements after those tables are built, as we should not change quartz code, right? In this way, why not just have a big file, which just serves as a reference to users.

Reply:
I see. 
So you are ok with breaking the database setup code then?
Could you come up with a plan for how to evolve DB schemas?

Reply:
What do you mean by breaking database setup code? I believe new DatabaseSetup should be able to deal with both the two cases: 1. Sql folders; 2. one Sql file.